Git-Rebase.md

```diff
@@ -35,7 +37,7 @@ The repo will now look like this:
 Do you see what happened? Git essentially saved the commits in the `auth` branch, 'removed' it, and then created it again with the same commits *after* the commits in the `dev` branch. This means that `Commit 4` only exists in the `dev` branch and not the `auth` branch! And that is really all there is to it! This might seem a bit confusing at first, but try to understand the diagram. This is an extremely useful tool.
 
 
-## git-rebase at FCC
+## Git-Rebase at FCC

```

Comment:
Something like this would make a perma link
```md
## <a id="fcc-git-rebase"></a>Git-Rebase at FCC
```
which will appear as...

## <a id="fcc-git-rebase"></a>Git-Rebase at FCC